.github/workflows/actionlint.yml

```diff
@@ -7,14 +7,14 @@ on:
       - "main"
       - "release-**"
     paths:
-      - '.github/workflows/*.ya?ml'
+      - '.github/**/*.ya?ml'
```

Comment:
Not sure why this change. Workflows are always in .github/workflows. So widening this for actionlint is wrong.

Reply:
And this is not a regex. :-)

Reply:
`actions/action.yml`?

Reply:
`actions` holds a composite workflow (this is new since I last looked :-). So that could be `.github/actions/**/*.ya?ml` then.

Reply:
Ideally, that action could live in the `workflows` dir... @russellb can I move it there? or does it need to be under an `actions/` directory?

Reply:
I think it is common convention to put composite actions under `.github/actions` but I don't believe it is required.

Reply:
i think i followed docs to put it there

and I thought the changed looked appropriate so it found changes under actions/ as well

Reply:
I went with what I hope is a happy medium here